Rubberduck.Core/UI/Inspections/ComponentTypeConverter.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+using Rubberduck.VBEditor.SafeComWrappers;
+
+namespace Rubberduck.UI.Inspections
+{
+    class ComponentTypeConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is ComponentType componentType)
+            {
+                switch (componentType)
+                {
+                    case ComponentType.ComComponent:
+                        return Resources.RubberduckUI.CodeInspections_ComponentType_ComComponent;
+                    case ComponentType.Undefined:
```

Comment:
You could actually make this an extensionMethod `ToLocalizedString`.
Moreover, although this is actually the cleaner way to do it, you could do what is done in the `DeclarationTypeExtensions`:
```
RubberduckUI.ResourceManager.GetString("DeclarationType_" + type, CultureInfo.CurrentUICulture);
```

Reply:
Thanks for bringing the extension method option to my attention. I tried creating it but that caused a circular dependency.